commit 6dff095c7f6aba04f3e3dc43a88f6c96ee3d25e4
Author: Martin Koller <kollix@aon.at>
Date:   Fri Aug 3 13:39:38 2012 +0200

    Fix for: When a new job is shown in the notification applet, initially the title is unset
    
    BUG: 288093
    FIXED-IN: 4.9.1

diff --git a/plasma/generic/applets/notifications/ui/jobwidget.cpp b/plasma/generic/applets/notifications/ui/jobwidget.cpp
index 02371c0..06096fd 100644
--- a/plasma/generic/applets/notifications/ui/jobwidget.cpp
+++ b/plasma/generic/applets/notifications/ui/jobwidget.cpp
@@ -146,6 +146,7 @@ JobWidget::JobWidget(Job *job, Plasma::ExtenderItem *parent)
         connect(stopAction, SIGNAL(triggered()), m_job.data(), SLOT(stop()));
 
         updateJob();
+        updateJobState();  // make sure to set the title
     } else {
         m_extenderItem->showCloseButton();
 
